htmd/parameterization/ffmolecule.py

```diff
@@ -293,28 +293,43 @@ def fitDihedrals(self, dihedrals, geomopt=True):
 
             molecules.append(mol)
 
-        # Run QM calculation of the rotamers
-        dirname = 'dihedral-opt' if geomopt else 'dihedral-single-point'
-        qm_results = []
-        for dihedral, molecule in zip(dihedrals, molecules):
-            name = "%s-%s-%s-%s" % tuple(self.name[dihedral])
-            fitdir = os.path.join(self.outdir, dirname, name, self.output_directory_name())
-            os.makedirs(fitdir, exist_ok=True)
+        # Create directories for QM data
+        directories = []
+        dihedral_directory = 'dihedral-opt' if geomopt else 'dihedral-single-point'
+        for dihedral in dihedrals:
+            dihedral_name = '%s-%s-%s-%s' % tuple(self.name[dihedral])
```

Comment:
Try to use .format() for formatting strings. More python3-style. The % formatting is considered old-style.